packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2889,20 +2889,33 @@ int _toLocationId(_Location location) {
   return id;
 }
 
-class _SerializationDelegate implements DiagnosticsSerializationDelegate {
-  _SerializationDelegate({
+/// A delegate that configures how a hierarchy of [DiagnosticsNode]s should be
```

Comment:
Add a `@visibleForTesting` annotation.

Tweak the comment to
/// A delegate that configures how a hierarchy of [DiagnosticsNode]s are
/// serialized by the Flutter Inspector.